hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -690,6 +690,7 @@ private Result increment(final HRegion region, final OperationQuota quota,
     if (metricsRegionServer != null) {
       long blockBytesScanned =
         context != null ? context.getBlockBytesScanned() - blockBytesScannedBefore : 0;
+      quota.addBlockBytesScanned(blockBytesScanned);
```

Comment:
this may not be a huge issue, but i think this shouldn't be within the metricsRegionServer != null check. Maybe move this and the above line up outside

Reply:
Also you missed adding this to `append`